### PR TITLE
Contributing links

### PR DESCRIPTION
**Pull request by jburel**
Update links to properties files

### CODE REVIEW

File: contributing/schema-changes.rst [hunk 1]
```diff
@@ -73,10 +73,11 @@ the :literal:`psql` shell:
         (1 row)
 
 indicating that a database is on patch version 4. Correspondingly,
+in :omero_subs_github_repo_root:`omero-model`,
 
 ::
 
-        $ grep ^omero.db.patch= etc/omero.properties
+        $ grep ^omero.db.patch= src/main/java/resources/omero-model.properties
```
Comment (mtbc) [inline contributing/schema-changes.rst]: Shouldn't have the `java/` in there. (Also below.)

File: contributing/schema-changes.rst [hunk 2]
```diff
@@ -140,5 +144,5 @@ diff` reports should help you to create a new
 :file:`sql/psql/OMERO5.1DEV__5/OMERO5.1DEV__4.sql`.
 
 When you commit your code and issue a pull request, include the
-changes to :omero_source:`omero.properties <etc/omero.properties>` and
+changes to :omero_model_source:`omero-model.properties <src/main/resources/omero-model.properties>` and
```
Comment (mtbc) [inline contributing/schema-changes.rst:147]: Outwith this PR, in the longer term I wonder if `build-schema` should just be part of the normal build and release process so it goes in the zip but doesn't have the autogen output in version control.

Comment (joshmoore) [reply contributing/schema-changes.rst:147]: > but doesn't have the autogen output in version control.

No strong objection. There are times where I've `git grep`'d or `git log`'d on that directory, but that can be worked around.

Comment (jburel) [reply contributing/schema-changes.rst:147]: build-schema needs to be moved "down" in the stack. As discussed we do not need it for 5.5, but one thing  to look after.

**Review by mtbc — APPROVED** [review]

**Pull request merged**